### PR TITLE
Karina 42 mobile display

### LINKED ISSUE #42: Improve Mobile Display

### PR DESCRIPTION
**Pull request by jinbest17**

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -39,6 +39,7 @@ <h1 id="pomobear-header">Pomobear</h1>
         >
         </button>
     </div>
+    <img id="pomo-logo" src="./images/logo.svg" alt="Pomobear Logo" width="50">
```
Comment (cbj99) [inline source/index.html:42]: good, logo.svg added

File: source/scripts/accessibility.js [hunk 2]
```diff
@@ -51,15 +51,19 @@ function keyControls (e) {
...
-    case keys.LEFT_ARROW:
-      e.preventDefault();
-      removeAll();
-      (settingsPaneIsOpen) ? closeSettingsPane() : openStatsPane();
-      break;
     case keys.RIGHT_ARROW:
-      e.preventDefault();
-      removeAll();
-      (statsPaneIsOpen) ? closeStatsPane() : openSettingsPane();
+      if(!statsButton.disabled) {
```
Comment (cbj99) [inline source/scripts/accessibility.js]: good, fixing the keyboard access

File: source/styles/main.css [hunk 3]
```diff
@@ -91,12 +91,20 @@ main {
 
 #pomobear-header-container {
     position: absolute;
-    left: 5%;
-    font-size: 50px;
+    left: 30px;
```
Comment (cbj99) [inline source/styles/main.css:94]: Could be refactored into constants instead of current "magic numbers"

**Review by hod002 — COMMENTED** [review]
Pane tests need to be fixed

**Review by hod002 — COMMENTED** [review]
SpecTests keystroke.spec failing

**Review by cbj99 — APPROVED** [review]
Good changes, some possible refactoring specified in comments, overall good work!

**Review by hod002 — APPROVED** [review]

**Pull request merged**